src/compiler/parser.ts

```diff
@@ -9118,7 +9119,7 @@ namespace Parser {
 
             function parseReturnTag(start: number, tagName: Identifier, indent: number, indentText: string): JSDocReturnTag {
                 if (some(tags, isJSDocReturnTag)) {
-                    parseErrorAt(tagName.pos, scanner.getTokenStart(), Diagnostics._0_tag_already_specified, tagName.escapedText);
+                    parseErrorAt(tagName.pos, scanner.getTokenStart(), Diagnostics._0_tag_already_specified, unescapeLeadingUnderscores(tagName.escapedText));

```

Comment:
Here's one example of accidentally using an escaped name in an error message.

Reply:
True - though it's always going to be the text `return` anyhow.

Reply:
This is probably a bad example, yeah, it's just the first one shown in the unhidden diff.